test/integration/scan_test.go

```diff
@@ -355,13 +370,15 @@ func TestScanCreateWithThresholdAndReportGenerate(t *testing.T) {
 		flag(params.TargetFormatFlag), "json",
 		flag(params.TargetPathFlag), "/tmp/",
 		flag(params.TargetFlag), "results",
+		flag(params.AstAPIKeyFlag), originals[params.AstAPIKeyEnv],
 	}
 
 	cmd := createASTIntegrationTestCommand(t)
 	err := executeWithTimeout(cmd, 5*time.Minute, args...)
 	assertError(t, err, "Threshold check finished with status Failed")
 
-	_, fileError := os.Stat(fmt.Sprintf("%s%s.%s", "/tmp/", "results", "json"))
+	file, fileError := os.Stat(fmt.Sprintf("%s%s.%s", "/tmp/", "results", "json"))
+	assert.Assert(t, file.Size() > 5000, "Report file should be bigger than 5KB")
```

Comment:
Report file **mast** be bigger than 5KB??

Reply:
This is an assertion that Or told me to add

Reply:
@tamarleviCm , this is a new assertion that I asked to add that will verify the download was completed successfully with a good report